DEVELOPMENT.md

```diff
@@ -32,16 +32,15 @@ It contains instructions to build, run, and test Kyverno.
 - Clone the project to your local machine.
 - Make sure that you have the Visual Studio Code editor installed on your system.
 
-- Make sure that you have the Docker installed on your system.
+- Make sure that you have wsl(Ubuntu preferred) and Docker installed on your system and on wsl too (docker.sock (UNIX socket) file is necessary to enable devcontainer to communicate with docker running in host machine).
```

Comment:
Is this only for Ubuntu? I'd like to give it try on MAC.

Reply:
No, Unix-based systems like Linux are the only ones that support the "/var/run/docker.sock" file. The Docker daemon communicates with the Docker API using this as its default Unix socket. A lightweight Linux virtual machine houses the Docker daemon on macOS, where it communicates via a different method.

Reply:
@realshuting wanna discuss it more ?